android/src/main/java/com/RNAppleAuthentication/SignInWithAppleConfiguration.kt

```diff
@@ -7,7 +7,8 @@ data class SignInWithAppleConfiguration private constructor(
   val responseType: String,
   val state: String,
   val rawNonce: String,
-  val nonce: String
+  val nonce: String,
+  val fullScreen: Boolean = false
```

Comment:
documented as defaulting to true, and in other spots has nullish-coalescing default to true, but here is false default

```suggestion
  val fullScreen: Boolean = true
```

Reply:
☑️